app/containers/CreateWallet/CreateWallet.scss

```diff
@@ -27,3 +27,9 @@
   margin-top: auto;
   margin-bottom: 50px;
 }
+
+.SelectContainer {
```

Comment:
For consistency can we stick with the pattern of having class names `camelCased`

Reply:
Thanks for the advice! I will keep in mind going forward. For this particular case, I do not need `SelectContainer` anymore because I will use Tabs component as you have suggested below.